FakeServer/Controllers/DynamicController.cs

```diff
@@ -54,20 +54,23 @@ public void UpdateAllData([FromBody]string value)
         /// <remarks>
         /// Add filtering with query parameters. E.q. /api/users?age=22&amp;name=Phil (not possible with Swagger)
         ///
-        /// Optional parameter names skip/take and offset/limit:
+        /// Optional parameter names skip/take and offset/limit and page/per_page:
         /// /api/users?skip=10&amp;take=20
         /// /api/users?offset=10&amp;limit=20
-        /// </remarks>
+        /// /api/users?page=2&amp;per_page=20
+        ///  </remarks>
         /// <param name="collectionId">Collection id</param>
         /// <param name="skip">Items to skip (optional name offset)</param>
         /// <param name="take">Items to take (optional name limit)</param>
+        /// <param name="page">Page number of items to get</param>
+        /// <param name="per_page">Number of items to get per page</param>
         /// <returns>List of items</returns>
         /// <response code="200">Collection item array</response>
         /// <response code="400">Invalid query parameters</response>
         /// <response code="404">Collection not found</response>
         [HttpGet("{collectionId}")]
         [HttpHead("{collectionId}")]
-        public IActionResult GetItems(string collectionId, int skip = 0, int take = 512)
+        public IActionResult GetItems(string collectionId, int skip = 0, int take = 512, int? page = null, int? per_page = null)
```

Comment:
Remove page and per_page from method parameters and move handling to QueryHelper.GetQueryOptions (same way as offset/limit). Method has a skip and take with default values, so it is easier to call it with Swagger. Those are also most common used parameters.

Reply:
Oh I see now. I removed the page and per_page from DynamicController and moved the handling of them into QueryHelper as you suggested.